tbump/file_bumper.py

```diff
@@ -275,7 +333,7 @@ def compute_change_request_for_file(self, file: tbump.config.File) -> ChangeRequ
 
         to_search = None
         if file.search:
-            to_search = file.search.format(current_version=re.escape(current_version))
```

Comment:
This one looked like redundant to me, as well as blocked using the `current_version` later on in regex expressions correctly.

Reply:
I think you're right. Could you make a separate commit/PR so we can see if this is true?